src/lib/source-handlers/azure/list-projects.ts

```diff
@@ -0,0 +1,116 @@
+import { AzureProjectData as Project } from './types';
+import base64 = require('base-64');
+import Bottleneck from 'bottleneck';
+import * as debugLib from 'debug';
+import { AzureProjectData } from './types';
+import { getAzureToken } from './get-azure-token';
+import * as needle from 'needle';
+
+const debug = debugLib('snyk:azure');
+const azureDefaultUrl = 'https://dev.azure.com';
```

Comment:
this has to be passed in by the user as an option to the CLI `sourceUrl`, see my previous comment on this. The default can be `https://dev.azure.com` but if a custom URL is needed user must set `sourceUrl`

Reply:
But sourceUrl is not a required param, I only apply this default url when no param was given

Reply:
Done